Some changes for 757-200

frank-deng opened the pull request:
Changes I have made:
-Added sound effect when the viewport is inside cabin.
-Fixed the sound effect for Cabin View.
-Cabin Visibility is set to No by default to reduce GPU usage.
-Vertical speed for Flight Level Change mode changed to 1000ft/min and -1000ft/min
-Automatically refuel the aircraft based on the flight plan specified on startup.

frank-deng committed:
updated

frank-deng committed:
updated_changes

dogsaysmoo commented:
Hi frank-deng,

Please see my comments on the 747-8i about the auto-refuelling.

I would rather leave the default for the cabin model visibility the same, but instead make the setting save between sessions so you only have to turn it off once.

When I have a chance, I'll review the sound changes.  That's looking pretty promising.  Thanks.

frank-deng committed:
Use /sim/fuel-fraction as initial fuel amount. Automatic fuel redistribution on startup

frank-deng committed:
Cockpit displays by default

frank-deng committed:
Cabin visibility saved between sessions

frank-deng commented:
Hi,
The newest changes are as follows based on the base branch:
-Added sound effect when the viewport is inside cabin.
-Fixed the sound effect for Cabin View.
-Cabin Visibility is saved between sessions.
-Vertical speed for Flight Level Change mode changed to 1000ft/min and -1000ft/min
-Automatically redistribute fuel between tanks on startup, the initial fuel amount is specified by /sim/fuel-fraction.

frank-deng committed:
Updated fuel_system

frank-deng committed:
updated sound

dogsaysmoo commented:
Merged! Thanks, Frank.  I like the new interior engine sounds a lot.  Sounds more real.
